### DISCUSSION TITLE
exception eaccessviolation in module laragon.exe at 0015AB79

### DISCUSSION THREAD

**Discussion by HawkbloodStudio** [leokhoa/laragon#401] [Q&A]
with no reason, my laragon just didn't work and show this notification:

exception EAccessViolation in module laragon.exe at 0015AB79. Access Violation at address 0055AB79 in module 'laragon.exe'. Read of address 00000B30.

I try to unninstall it (2020 download), and install the new one using new download instalation (now/2023), but it still appear. 

is there a way to solve this?

**Comment by leokhoa** [comment]
@HawkbloodStudio 
Did you update Windows recently?
Normally next update of Windows will fix that.

**Reply by HawkbloodStudio** [reply]
yes, I never skip the latest update. btw, it appear after the last update of windows

**Reply by psycongaroo** [reply]
> yes, I never skip the latest update. btw, it appear after the last update of windows

did u solved it?

**Comment by psycongaroo** [comment]
Having same problem on Windows Server 2019-2022.

**Reply by rusmanplatd** [reply]
same problem here on windows 11 home latest update

**Reply by maulvi** [reply]
same here on Windows Server 2019

**Comment by maulvi** [comment]
> same here on Windows Server 2019

portable version working fine but the full version throws error
